### PR TITLE
add violations count on cli output (#224)

### PR DESCRIPTION
**Pull request by ricfio**

### CODE REVIEW

**Comment by AlessandroMinoccheri** [comment]
Why the PR is in draft status? What does it miss @ricfio?

**Comment by ricfio** [comment]
> Why the PR is in draft status? What does it miss @ricfio?

Because I have a pending question about the row 146 in `src/CLI/Check.php`.
Can you see my related comment on the 'Files changed' page?

image

In truth it is not a comment... I'm not sure this is the right way...
I have added a 'code review' request for my self?

**Comment by AlessandroMinoccheri** [comment]
@ricfio I can't see your comment, but in my opinion, you can try to write the feature, and after we can discuss it together :)

File: src/CLI/Check.php [hunk 1]
```diff
@@ -143,8 +143,8 @@ private function getConfigFilename(InputInterface $input)
 
     private function printViolations(Violations $violations, OutputInterface $output): void
     {
-        $output->writeln('<error>ERRORS!</error>');
         $output->writeln(sprintf('%s', $violations->toString()));
```
Comment (AlessandroMinoccheri) [inline src/CLI/Check.php:146]: In my opinion, we can keep the `ERRORS` line when we start printing all the violations

**Review by AlessandroMinoccheri — CHANGES_REQUESTED** [review]

File: src/CLI/Check.php (hunk 1 above)
Comment (ricfio) [reply src/CLI/Check.php:146]: I think the `ERRORS` string is superfluous and redundant at this point, are you sure you want hold it?
In this case, do you prefer rollback also the tests (modified with my last commit) based on `ERRORS` string?

Comment (AlessandroMinoccheri) [reply src/CLI/Check.php:146]: We can try to remove it for now and understand later if it was necessary or more clear with it @ricfio

**Comment by ricfio** [comment]
Ok, I have removed the last commit on this branch.
The new PR is here: https://github.com/phparkitect/arkitect/pull/237

**Pull request closed without merging**